heft-plugins/heft-browsersync-plugin/schemas/browsersync.schema.json

```
@@ -0,0 +1,29 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Browsersync Configuration",
+  "description": "Defines optional options for Browsersync plugin configuration",
+  "type": "object",
+  "additionalProperties": false,
+  "properties": {
+    "$schema": {
+      "description": "Part of the JSON Schema standard, this optional keyword declares the URL of the schema that the file conforms to. Editors may download the schema and use it to perform syntax highlighting.",
+      "type": "string"
+    },
+    "extends": {
+      "description": "Optionally specifies another JSON config file that this file extends from.  This provides a way for standard settings to be shared across multiple projects.",
+      "type": "string"
+    },
+    "rootDir": {
```

```suggestion
    "serveRoot": {
```

Is this not just the project folder?

---

It depends on project structure so it definitely needs to be configurable.